integral_view/views/system_date_time.py

```diff
@@ -0,0 +1,44 @@
+import django
+import django.template
+
+from integral_view.forms import date_time_form
+from integralstor.update_system_date_time import update_system_date_time
+
+
+def view_date_time(request):
```

Comment:
Why do we need this view at all? We already have a screen that shows the date and time..